app/beacon-omop-worker/beacon_omop_worker/beacon_dto/filtering_term.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+from beacon_omop_worker.beacon_dto.base_dto import BaseDto
+
+
+class FilteringTerm(BaseDto):
```

Comment:
Just a note that if you have more DTO like objects and serialization to come, you may want to look at Pydantic to make this nicer